src/io/flutter/run/daemon/FlutterApp.java

```diff
@@ -83,6 +84,8 @@
   private int userReloadCount;
   private int restartCount;
 
+  private long maxFileTimestamp;
```

Comment:
What I needed here was a way to tell that files are out of date from the previous run/hot reload. If there is a way to detect that more simply let me know.